### PR TITLE
Fix status display for 'Publish Sign Off' field in Superdesk

### PR DESCRIPTION
**Pull request by dzonidoo**
TGA-111

### CODE REVIEW

File: client/extensions/tga-sign-off/src/components/SignOffListItem.tsx [hunk 1]
```diff
@@ -34,6 +34,14 @@ type IProps = IPropsApproved | IPropsPendingOrExpired | IPropsNotSend;
 export function SignOffListItem(props: IProps) {
     const {formatDateTime, gettext} = superdesk.localization;
 
+    const isExpired = (date: string) => {
```
Comment (tomaskikutis) [inline client/extensions/tga-sign-off/src/components/SignOffListItem.tsx]: move this one out of `SignOffListItem` function so it is clear it is not using anything from parent scope.

**Comment by tomaskikutis** [comment]
@petrjasek I'm thinking this solution isn't the most stable one. Could we hit some endpoint and get status from there? Duplicating the logic in frontend/backend that computes the status isn't good.

**Comment by petrjasek** [comment]
there should be also `expires` timestamp field in the request, at least from what I see in the server code

**Comment by tomaskikutis** [comment]
I see. @dzonidoo then use the expires date rather assuming it's 24h, so if we want to change in the future to say 48h it'd be enough to update the server.

File: client/extensions/tga-sign-off/src/components/SignOffListItem.tsx [hunk 2]
```diff
@@ -32,9 +33,8 @@ interface IPropsNotSend extends IPropsBase {
 type IProps = IPropsApproved | IPropsPendingOrExpired | IPropsNotSend;
 
 const isExpired = (date: string) => {
-    const sentDate = new Date(date);
     const currentDate = new Date();
-    const expiryDate = new Date(sentDate.getTime() + 24 * 60 * 60 * 1000);
+    const expiryDate = new Date(date);
```
Comment (tomaskikutis) [inline client/extensions/tga-sign-off/src/components/SignOffListItem.tsx:37]: simplify it to be `const isExpired = new Date() > new Date(props.expiryDate)`

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**